binder/cbor.go

```diff
@@ -0,0 +1,15 @@
+package binder
+
+import (
+	"github.com/gofiber/utils/v2"
+)
+
+type cborBinding struct{}
+
+func (*cborBinding) Name() string {
```

Comment:
This whole file is missing tests

Reply:
> This whole file is missing tests

None of the binders in bind/ folder is tested. I suppose we have to create separate test-cases for them since codecov shows lower coverage. I can create another PR for that

Reply:
Agree